### PR TITLE
intellij warnings cleared

### LINKED ISSUE #23: [Misc 3] Fix IntelliJ warnings
We should go through all the files once we have a working project and do a check of all files that warning have been addressed.

### PR DESCRIPTION
**Pull request by SMateen21**
title

### CODE REVIEW

**Review by lilyxmeng — APPROVED** [review]

**Review by nyan-ko — APPROVED** [review]
thanks!

**Review by meiiph — APPROVED** [review]

**Comment by SMateen21** [comment]
I sent this in before the deadline but didnt get approval due to ReadMe issues

**Pull request merged**
